lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -73,7 +73,7 @@ def capture(money, authorization, options={})
         post = {}
         add_integration_key(post)
         post[:hash] = authorization
-        post[:amount] = amount(money)
+        post[:amount] = amount(money) if customer_country(options) == 'br'
```

Comment:
The drawback is if the merchant never intends to transact partial captures in Brazil, the transaction will always fail...thus requiring the merchant to contact the Ebanx Integration Team to enable partial captures.

Reply:
I wonder if rather than checking country and always adding in that case, it makes sense to add a flag so that the merchant can specify whether or not they want to do a partial capture to avoid instances where it may fail unexpectedly? looks like we do something similar for bluesnap: https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/blue_snap.rb#L98

then we could just document the circumstances where partial captures work. what do you think?

Reply:
@britth That sounds like a much better approach. I'll look into the bluesnap implementation. Thanks!